webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/JsonSchema.scala

```diff
@@ -0,0 +1,1220 @@
+/*
```

Comment:
This file needs to be splitted. I see 4 files (at least)

* JsonResponse.scala: containing everything related to making a response (abstract and lift one)
* JsonResponseObjects.scala: containing all classes that represents our domain andsend as response(this file will be sponsored by JR Ewing from 'Dallas' TV soap )
* JsonEncoders.scala: containing all data encoders
* JsonQueryObjects: containg all classe that represents all data received for our queries

Reply:
I'm not sure about the advantage of all being in the same file, but if you think so, we will see :)